### ISSUE TITLE
Timeout slider

### ISSUE THREAD

**Issue by M1Aston** [vanilla-music/vanilla#843]
- Vanilla Music Version (Settings -> About):1.0.72
- Android Version:8
- Phone vendor / model: XPeria X
- Application was downloaded from (use `[x]`):
  - [ ] Play Store
  - [ x] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [ ] Yes
  - [ ] No
  - [x ] NA / Don't know.

## Description

Is it just me, or is it really difficult to set the right timeout with the slider? I find it very fiddly (if not impossible) to set exactly the right minutes\seconds. The small slider is awkward to operate.

**Comment by andiandi13** [comment]
What do you mean ? I find it quite accurate, and you can long press and slide to seek more precisely.

**Comment by M1Aston** [comment]
Yes, I know. But I find it troublesome to set the exact time. Operating such a small slider to set to the exact second... well I think it's not exactly the most convenient method and difficult.

**Comment by andiandi13** [comment]
I would agree for very long files, and that's why I created #827

**Comment by breversa** [comment]
I agree with @M1Aston : the inactivity timeout slider in Settings/Misc is really awkward to use. There's no way you can set it to 20 minutes (for instance) using your fingers.

You'll always end up with either something like 19'23", or 20'18". Not that it's a big deal in itself, but it feels frustrating everytime. :( Also, does anyone need down-to-the-second precision for an inactivity timeout ? :) I'd be perfectly fine with minutes only.

**Comment by M1Aston** [comment]
All of the above... and the lack of a sleep timer, or at the very least an option to stop playback after x number of tracks and\or currently playing.
